### ISSUE TITLE
"the" solution

### ISSUE THREAD

**Issue by quist00** [carpentries/workbench#83]
Challenge solution banner reads "Show me the solution"

Recommend change to "Show a solution"

**Comment by tobyhodges** [comment]
I agree @quist00. I believe this change could be made by [modifying the `lesson.lua` file in `sandpaper`](https://github.com/carpentries/sandpaper/blob/bfb24a50495711da0f8dfeb48e4af33823451df4/inst/rmarkdown/lua/lesson.lua#L213). Likely that it would also mean updating a few tests, which will expect to see the old version of the heading in the test output.

**Comment by froggleston** [comment]
Unfortunately this isn't quite as simple, because this is a translatable string and would therefore require a rebuild of all language files to support the new wording. It is possible, but I would argue this is a low priority at present.
